grafast/dataplan-pg/src/codecs.ts

```diff
@@ -112,7 +161,7 @@ export interface PgCodecAttribute<
    * these are all plural relationships. So identicalVia is generally one-way
    * (except in 1-to-1 relationships).
    */
-  identicalVia?: PgCodecAttributeVia;
+  identicalVia?: _AnyPgCodecAttributeVia;
```

Comment:
If I change `via` and `identicalVia` to `GenericPgCodecAttributeVia` then I get weird errors in `exampleSchema.ts`. However, I know that if `via` is set then it will always be `string | { relation: string, attribute: string }` (or a narrower form of this) so I don't really understand why this needs to use `any`s?